### PR TITLE
PEP8 fixes

### PR DESCRIPTION
**Pull request by davidmcclure**
This clears PEP8 violations flagged by `flake8` -- mostly just line lengths.

### CODE REVIEW

File: osp_pipeline/corpus.py [hunk 1]
```diff
@@ -81,34 +113,53 @@ def parse_http_response(b):
...
     :rtype: Spark dataframe
     """
     rdd = spark.sparkContext.emptyRDD()
+
     for p in paths:
         rdd = rdd.union(spark.sparkContext.binaryFiles(p))
-    warcs = rdd.flatMap(lambda t: try_or_log2(extract_from_warc(BytesIO(t[1]), kinds={'response'}))).filter(not_none)
-    rows = warcs.map(lambda warc: try_or_log(warc_to_row)(warc)).filter(not_none)
+
+    # TODO: How to format these deeply nested lambdas?
```
Comment (wearpants) [inline osp_pipeline/corpus.py]: I played around with a few formatting options & didn't like any of them either. :-[

Comment (wearpants) [inline osp_pipeline/corpus.py]: Maybe line continuations `\` would be better than parens around the whole thing? I've certainly seen that before, and the bare-looking `rdd` confused me.

File: osp_pipeline/corpus.py [hunk 2]
```diff
@@ -117,24 +168,39 @@ def warcs_to_df(spark, paths):
...
     :arg sc: Spark context
-    :arg blob: classifier binary blob. This will be be broadcast across the cluster.
+    :arg blob: Classifier binary blob.
     :arg texts: RDD of (id, text)
     :rtype: RDD of (id, score)
     """
     bcast = sc.broadcast(blob)
-    return texts.map(lambda t: t[0], try_or_log(classify)(bcast.value, t[1])).filter(not_none2)
+
+    # TODO: Where does `classify` come from?
```
Comment (wearpants) [inline osp_pipeline/corpus.py]: Needs to be written :-)

File: osp_pipeline/corpus.py (hunk 1 above)
Comment (davidmcclure) [inline osp_pipeline/corpus.py]: Yeah, agree about `rdd` alone on the line. An alternative could be to put the first chained method call on the same line as the instance, and then start breaking after that:

```python
warcs = (
    rdd.flatMap(lambda t: try_or_log2(
        extract_from_warc(BytesIO(t[1]), kinds={'response'})
    ))
    .filter(not_none)
)
```

Comment (wearpants) [inline osp_pipeline/corpus.py]: I was thinking something more like this (which is basically what you have without the outer parens):

```python
warcs = rdd.flatMap(lambda t: try_or_log2(
    extract_from_warc(BytesIO(t[1]), kinds={'response'})
    )).filter(not_none)
```

Comment (davidmcclure) [inline osp_pipeline/corpus.py]: What if we want to keep chaining though? Gets messy:

```python
warcs = rdd.flatMap(lambda t: try_or_log2(
    extract_from_warc(BytesIO(t[1]), kinds={'response'}))
    ).filter(not_none).flatMap(lambda t: try_or_log(something_else(t)).filter(not_none)
```

The advantage to the outer parens is that it allows the chain to grow vertically, without needing the line continuations:

```python
warcs = (
    rdd.flatMap(lambda t: try_or_log2(
        extract_from_warc(BytesIO(t[1]), kinds={'response'})
    ))
    .filter(not_none)
    .flatMap(lambda t: try_or_log2(
        something_else(t)
    ))
    .filter(not_none)
)
```

Doesn't really matter though -- I think it's nice to keep the lines under 80 chars, but I don't feel strongly beyond that. Let's just see what feels good as we get into the gnarlier jobs?

**Comment by davidmcclure** [comment]
Pushed a commit that fixes the rdd instance variables on separate lines in the chained expressions. This should be good to go?

**Pull request merged**